lib/services/LeaderElectionService.ts

```diff
@@ -55,10 +55,14 @@ export class LeaderElectionService implements ServiceInterface {
     await this.stop();
     if (this.canStart()) {
```

Comment:
maybe `canStart` should evaluate `this.started` as well

Reply:
Added check of `this.started`